src/includes/class-alma-wc-share-of-checkout-helper.php

```diff
@@ -0,0 +1,237 @@
+<?php
+/**
+ * Alma share of checkout helper
+ *
+ * @package Alma_WooCommerce_Gateway
+ */
+
+use Alma\API\RequestError;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	die( 'Not allowed' ); // Exit if accessed directly.
+}
+
+/**
+ * Alma_WC_Share_Of_Checkout_Helper
+ */
+class Alma_WC_Share_Of_Checkout_Helper {
+
+	/**
+	 * Logger
+	 *
+	 * @var Alma_WC_Logger
+	 */
+	private $logger;
+
+	/**
+	 * Start time;
+	 *
+	 * @var string
+	 */
+	private $start_time;
+
+	/**
+	 * End time;
+	 *
+	 * @var string
+	 */
+	private $end_time;
+
+	/**
+	 * __construct.
+	 */
+	public function __construct() {
+		$this->logger     = new Alma_WC_Logger();
+		$this->start_time = null;
+		$this->end_time   = null;
+	}
+
+	/**
+	 * Set share of checkout "from date". (date BEGIN)
+	 *
+	 * @param string $start_time The start time.
+	 * @return void
+	 */
+	public function set_share_of_checkout_from_date( $start_time ) {
```

Comment:
as we are in the share of checkout helper, maybe we could have a shorter name ?
`set_share_of_checkout_from_date` -> `set_from_date`
`set_share_of_checkout_to_date` -> `set_to_date`